src/posit/connect/hooks.py

```diff
@@ -6,7 +6,7 @@
 from .errors import ClientError
 
 
-def handle_errors(response: Response, *args, **kwargs) -> Response:
+def handle_errors(response: Response, *args, **kwargs) -> Response:  # noqa: ARG001
```

Comment:
Is it possible to disable "ARG001" globally for *args and **kwargs only?

Reply:
I do not see a way.

Can I just remove the arguments?